js.go

```diff
@@ -984,6 +981,12 @@ func (js *js) subscribe(subj, queue string, cb MsgHandler, ch chan *Msg, isSync
 		if err != nil {
 			return nil, err
 		}
+
+		// With flow control enabled async subscriptions we will disable msgs limits,
+		// and set a larger pending bytes limit by default.
+		if cb != nil && hasFC {
+			sub.SetPendingLimits(-1, DefaultSubPendingBytesLimit*4)
```

Comment:
This would mean 256M max pending bytes for flow control based push consumers. Disabled the limit of number of messages but maybe could have done something similar and change this to *4: https://github.com/nats-io/nats.go/blob/aa4ab64c8ba0289099145ca795b6e51c90e8ca99/nats.go#L4119